Prerelease: v1.0.0-b2 (#892)
fishtown-analytics/docs.getdbt.com: website/docs/docs/contributing/building-a-new-adapter.md

</File>

There are a few things you can do to make it easier for users when connecting to your database:
- Be sure to implement the Credentials' `_connection_keys` method shown above. This method will return the keys that should be displayed in the output of the `dbt debug` command. As a general rule, it's good to return all the arguments used in connecting to the actual database except the password (even optional arguments).

> - Create a `profile_template.yml` that includes to prompt a brand-new user setting up a connection profile via the [`dbt init` command](init). See more details [below](#other-files).

Reviewer: ```suggestion
- Create a `profile_template.yml` to enable configuration prompts for a brand-new user setting up a connection profile via the [`dbt init` command](init). See more details [below](#other-files).
```
